wire-ios/Wire-iOS/Sources/Developer/DeveloperTools/SwitchBackend/SwitchBackendViewModel.swift

```diff
@@ -0,0 +1,93 @@
+//
+// Wire
+// Copyright (C) 2023 Wire Swiss GmbH
+//
+// This program is free software: you can redistribute it and/or modify
+// it under the terms of the GNU General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+//
+// This program is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+// GNU General Public License for more details.
+//
+// You should have received a copy of the GNU General Public License
+// along with this program. If not, see http://www.gnu.org/licenses/.
+//
+
+import Foundation
+import WireTransport
+
+@available(iOS 14, *)
+final class SwitchBackendViewModel: ObservableObject {
+
+    // MARK: - Models
+
+    struct Item: Identifiable {
+
+        let id = UUID()
+        let title: String
+        let value: EnvironmentType
+
+    }
+
+    enum Event {
+
+        case itemTapped(Item)
+
+    }
+
+    // MARK: - State
+
+    let items: [Item]
+
+    @Published
+    var selectedItemID: Item.ID
+
+    @Published
+    var isAlertPresented = false
+
+    @Published
+    var alertMessage = ""
```

Comment:
you can probably have just one variable to represent the alert and use `.alert(item:` instead of `.alert(isPresented:`

Reply:
Nice I didn't know about that